### PR TITLE
update nightly

### PR DESCRIPTION
**Pull request by fbielejec**
# Description

Bump nightly version to fix compilation errors

## Type of change

Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)

# Checklist:

- I have made neccessary updates to the Infrastructure

### CODE REVIEW

**Review by mike1729 — APPROVED** [review]

File: rust-toolchain [hunk 1]
```diff
@@ -1 +1 @@
-nightly-2022-05-23
+nightly-2022-08-12
```
Comment (DamianStraszak) [inline rust-toolchain:1]: Maybe something like that would be more `pro`? It then automatically installs the wasm target.
```
[toolchain]
channel = "nightly-2022-08-12"
targets = [ "wasm32-unknown-unknown" ]
```

**Review by timorl — APPROVED** [review]
Looks good, although if @DamianStraszak's comment works it would be great to have.

File: rust-toolchain (hunk 1 above)
Comment (fbielejec) [reply rust-toolchain:1]: I wrestled with it in a past, but the actions-rs/toolchain action does not understand toml and it's still in v1. Probably unmaintained too: https://github.com/actions-rs/toolchain/issues/216

Comment (DamianStraszak) [reply rust-toolchain:1]: Ok, fine, then you can ignore my comment.

**Pull request merged**